packages/@o3r/testing/src/core/playwright/angular-materials/autocomplete-material.ts

```diff
@@ -12,12 +12,12 @@ export class MatAutocomplete extends O3rElement implements MatAutocompleteProfil
   }
 
   /** @inheritdoc */
-  public async selectByValue(value: string) {
+  public async selectByValue(value: string, timeout = 5000) {
```

Comment:
Why setting a static default value instead of relying on the default one from Playwright?
(Same question for the other impl)

Reply:
because they set it at global level pretty high for other tests (due to calls etc.). I could only find a value that impacts all the timeout on the app or all the one linked to the navigation.
In the prior version we already had this kind of timeout. It is something we lost with the migration to the Locator.